etcdserver/etcdserverpb/rpc.proto

```diff
@@ -637,6 +638,8 @@ message AuthUserAddResponse {
 
 message AuthUserGetResponse {
   ResponseHeader header = 1;
+
+  repeated string roles = 2;
```

Comment:
i think it is just for making operators life simpler. So they do not have to issue multiple requests. But probably we can do this in etcdctl and keep RPC the same?